### PR TITLE
Fix for Windows to make sure to run the build command...

### PR DESCRIPTION
**Pull request by Kubik-Rubik**
... to generate AcceptanceTester

With the change the build command is also executed properly on a Windows machine.

**How to test**

Run the tests and check whether it works on a Windows and non-Windows machine without having called the build command manually first.

### CODE REVIEW

**Comment by therunnergit** [comment]
@Test works fine

**Comment by compojoom** [comment]
This works. 

What also works is to do:
```
$this->_exec('vendor\bin\codecept.bat build'); 
```

So we could do a test for windows and execute the above line and for everything else we can run the old line. 

I'm not sure which solution is better. If we have the bin folder with all those scripts, shouuld we use it, or should we directly call the codeception files as Viktor did? Ideas? Recommendations?

**Comment by javigomez** [comment]
@Kubik-Rubik I would go instead for the solution that proposes Daniel. Executing the bin file without php in front.

**Comment by Kubik-Rubik** [comment]
Okay, I will update the PR. Thanks!

File: RoboFile.php [hunk 1]
```diff
@@ -112,8 +113,8 @@ public function runTest($pathToTestFile = null, $suite = 'acceptance')
 	{
 		$this->runSelenium();
 
-		// Make sure to Run the Build Command to Generate AcceptanceTester
-		$this->_exec("php vendor/bin/codecept build");
+		// Make sure to tun the build command to generate AcceptanceTester
```
Comment (javigomez) [inline RoboFile.php]: tun-a :fish:

**Pull request merged**